core/math/math_funcs.h

```diff
@@ -220,8 +152,8 @@ class Math {
 	static _ALWAYS_INLINE_ bool is_finite(double p_val) { return isfinite(p_val); }
 	static _ALWAYS_INLINE_ bool is_finite(float p_val) { return isfinite(p_val); }
 
-	static _ALWAYS_INLINE_ double abs(double g) { return absd(g); }
-	static _ALWAYS_INLINE_ float abs(float g) { return absf(g); }
+	static _ALWAYS_INLINE_ double abs(double g) { return g == 0 ? 0.0 : (g < 0 ? -g : g); }
```

Comment:
What's the reason for this change? The new implementation would definitely generate worse bit code:

https://godbolt.org/z/hse3PWEe1

Reply:
I think this was a holdover from another PR which was trying to migrate the `typedefs.h` macros to here. Looks like `::abs` indeed the better option (`std::abs` would need `cmath`), so I'll just use that.